server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1655,125 +1367,28 @@ public int countConsumers(
         return poolDTOs;
     }
 
-    /**
-     * Retrieves a list of Subscriptions for an Owner
-     *
-     * @return a list of Subscription objects
-     * @httpcode 404
-     * @httpcode 200
-     */
-    @GET
-    @Consumes(MediaType.APPLICATION_JSON)
-    @Produces(MediaType.APPLICATION_JSON)
-    @Path("{owner_key}/subscriptions")
-    @ApiOperation(notes = "Retrieves a list of Subscriptions for an Owner", value = "List Subscriptions")
-    @ApiResponses({ @ApiResponse(code = 404, message = "Owner not found") })
-    public List<Subscription> getSubscriptions(@PathParam("owner_key") String ownerKey) {
+    @Override
+    public List<SubscriptionDTO> getOwnerSubscriptions(String ownerKey) {
         Owner owner = this.findOwnerByKey(ownerKey);
 
-        List<Subscription> subscriptions = new LinkedList<>();
+        List<SubscriptionDTO> subscriptions = new LinkedList<>();
 
         for (Pool pool : this.poolManager.listPoolsByOwner(owner).list()) {
-            SourceSubscription srcsub = pool.getSourceSubscription();
 
+            SourceSubscription srcsub = pool.getSourceSubscription();
             if (srcsub != null && "master".equalsIgnoreCase(srcsub.getSubscriptionSubKey())) {
-                subscriptions.add(this.poolManager.fabricateSubscriptionFromPool(pool));
+                subscriptions.add(this.translator.translate(
+                    this.poolManager.fabricateSubscriptionFromPool(pool),
```

Comment:
I'd have to go back and look, but did we add a DTO for Subscription? Subscription is already a DTO.

Additionally, rather than converting twice, since we only use ```fabricateSubscriptionFromPool``` in the resources to convert a pool to a subscription, we could/should just write a translator that explicitly handles the translation from Pool to Subscription[DTO] and use that instead.

Reply:
We added a SubscriptionDTO when porting the SubscriptionResource to OpenApi (and it looks like we also do the double conversion there). We can't help not redefining it since everything that is part of the API has to be in defined in the yaml file. I agree with creating a Pool to SubscriptionDTO translator, and getting rid of the SubscriptionTranslator and the fabricateSubscriptionFromPool method.

Reply:
This was updated as per the previous changes done in PR: https://github.com/candlepin/candlepin/pull/2674.

Reply:
Updated the code to translate from Pool to SubscriptionDTO.